cloud-sql/mysql/client-side-encryption/src/main/java/cloudsql/tink/EncryptAndInsertData.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cloudsql.tink;
+
+// [START cloud_sql_mysql_encrypt_insert]
+
+import com.google.crypto.tink.Aead;
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import java.security.GeneralSecurityException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.Date;
+import javax.sql.DataSource;
+
+public class EncryptAndInsertData {
+
+  public static void main(String[] args) throws GeneralSecurityException, SQLException {
+    // Saving credentials in environment variables is convenient, but not secure - consider a more
+    // secure solution such as Cloud Secret Manager to help keep secrets safe.
+    String dbUser = System.getenv("DB_USER"); // e.g. "root", "mysql"
+    String dbPass = System.getenv("DB_PASS");
+    String dbName = System.getenv("DB_NAME");
+    String cloudSqlConnectionName = System.getenv("CLOUD_SQL_CONNECTION_NAME");
+    String kmsUri = System.getenv("CLOUD_KMS_URI");
```

Comment:
nit: Can we add comment examples for all of these?

Reply:
Done.